multicluster/build/yamls/antrea-multicluster-leader-namespaced.yml

```diff
@@ -23,6 +23,12 @@ metadata:
   name: antrea-mc-controller-role
   namespace: antrea-multicluster
 rules:
+- apiGroups:
```

Comment:
You should add this part to file `config/overlay/leader-ns/role.yaml` and run `make manifests`, otherwise, this part will be removed next time when there is any manifest updates.